classes/metric/manager.php

```diff
@@ -73,17 +70,16 @@ public static function get_frequency_labels(): array {
             self::FREQ_12HOUR => get_string('twelve_hour', 'tool_cloudmetrics'),
             self::FREQ_DAY => get_string('one_day', 'tool_cloudmetrics'),
             self::FREQ_WEEK => get_string('one_week', 'tool_cloudmetrics'),
-            self::FREQ_FORNTIGHT => get_string('one_fortnight', 'tool_cloudmetrics'),
             self::FREQ_MONTH => get_string('one_month', 'tool_cloudmetrics'),
         ];
     }
 
     /**
-     * Gets all metrics installed on the system.Returns an associative array of all metrics installed.
+     * Gets all metrics installed on the system. Returns an associative array of all metrics installed.
      *
      * @return array An associative array of name => metric.
      */
-    public static function get_metrics(): array {
+    public static function get_metrics(bool $enabledonly = false): array {
```

Comment:
Why is this false by default? I would generally assume most could would only want to care about what is currently enabled.

Reply:
It started off that way, so default behaviour is unchanged.

Reply:
this is pre-prod code, what breaks if its toggled?

Reply:
Done